services/gateway/sources/version3/mappings/transaction.js

```diff
@@ -21,14 +21,14 @@ module.exports = {
 	height: '=,number',
 	nonce: '=,string',
 	block: {
-		id: 'blockID,string',
+		id: '=,string',
 		height: '=,number',
-		timestamp: 'unixTimestamp,number',
+		timestamp: '=,number',
 	},
 	sender: {
-		address: 'senderID,string',
-		publicKey: 'senderPublicKey,string',
-		username: '=,string',
+		address: '=,string',
+		publicKey: '=,string',
+		name: '=,string',
 	},
 	signatures: '=',
 	params: '=',
```

Comment:
We need to update `IncludePending` to `executionStatus`
```suggestion
	params: '=',
	executionStatus: '=,string',
```